include/zephyr/bluetooth/hci_types.h

```diff
@@ -276,8 +276,15 @@ struct bt_hci_cmd_hdr {
 #define BT_FEAT_LE_ISO(feat)            (BT_FEAT_LE_CIS(feat) | \
 					BT_FEAT_LE_BIS(feat))
 
-/* LE States */
-#define BT_LE_STATES_PER_CONN_ADV(states)     (states & 0x0000004000000000)
+/* LE States. See Core_v5.4, Vol 4, Part E, Section 7.6.27 */
+#define BT_LE_STATES_PER_CONN_ADV(states)     (states & BIT64_MASK(35))
+
+#if defined(CONFIG_BT_SCAN_AND_INITIATE_IN_PARALLEL)
+/* Passive/Active scanner can be run in parallel with initiator. */
+#define BT_LE_STATES_SCAN_INIT(states) (states & (BIT64_MASK(22) | BIT64_MASK(23)))
```

Comment:
If a controller would only support passive scanning + initiator, that would not work.
What about
```suggestion
#define BT_LE_STATES_SCAN_INIT(states) ((states & (BIT64_MASK(22) | BIT64_MASK(23))) == BIT64_MASK(22) | BIT64_MASK(23))
```